addons/portal/static/src/scss/portal.scss

```diff
@@ -359,6 +359,13 @@ img, .media_iframe_video, .o_image {
 
     .o_portal_sidebar_content {
         z-index: -1;
+        padding: map-get($spacers, 3);
+
+        @include media-breakpoint-up(lg) {
+            margin-top: $spacer * -4;
+            padding: 0;
+            padding-top : $spacer * 4;
+        }
```

Comment:
I guess it's OK. 
Maybe it needs some js magic to calculate the header/navbar height.